dbms/src/Functions/FunctionsStringSearch.cpp

```diff
@@ -1597,7 +1597,7 @@ void registerFunctionsStringSearch(FunctionFactory & factory)
     factory.registerFunction<FunctionReplaceOne>();
     factory.registerFunction<FunctionReplaceAll>();
     // factory.registerFunction<FunctionPosition>();
-    factory.registerFunction<FunctionPositionUTF8>();
+    // factory.registerFunction<FunctionPositionUTF8>();
```

Comment:
looks like `FunctionPositionUTF8` is already defined in somewhere? I think you need to remove the original `FunctionPositionUTF8`

Reply:
If you want to add a specific function, please give it a meaningfull name, it looks to me `fastPath` is the fast path for calculate for `position`

Reply:
> If you want to add a specific function, please give it a meaningfull name, it looks to me `fastPath` is the fast path for calculate the `position`

done